README.md

```diff
@@ -593,6 +594,21 @@ All query executions happen remotely from the **kdb VS Code extension** either a
 
   ![kdb results view](https://github.com/KxSystems/kx-vscode/blob/main/img/kdbview-results.png?raw=true)
 
+## AxLibraries
+
+The following features of [AxLibraries](https://code.kx.com/developer/libraries/) are integrated and can be used within the extension:
+
+- Grammar of Graphics
+- q static linter
```

Comment:
Do we need some more words on this, can it link to an existing section in the doc?